isis/src/apollo/apps/apolloremrx/apolloremrx.cpp

```diff
@@ -0,0 +1,134 @@
+/** This is free and unencumbered software released into the public domain.
+
+The authors of ISIS do not claim copyright on the contents of this file.
+For more details about the LICENSE terms and the AUTHORS, you will
+find files of those names at the top level of this repository. **/
+
+/* SPDX-License-Identifier: CC0-1.0 */
+
+#include "ProcessByTile.h"
+#include "SpecialPixel.h"
+#include "Pvl.h"
+#include "UserInterface.h"
+#include "Brick.h"
+#include "Apollo.h"
+#include <QString>
+#include <cstdlib>
+
+#include "apolloremrx.h"
+
+using namespace std;
+namespace Isis {
+
+  void cpy(Buffer &in, Buffer &out);
+  static int dim;
+  static bool resvalid;
+  static QString action;
+
+  void apolloremrx(UserInterface &ui) {
+      Cube *cube = new Cube(ui.GetFileName("FROM"), "rw");
+      apolloremrx(cube, ui);
+  }
+
+  void apolloremrx(Cube *info, UserInterface &ui) {
+    // We will be processing by line
+    ProcessByTile p;
+    p.SetTileSize(128, 128);
+
+    // Setup the input and output cubes
+    CubeAttributeInput &inputAtt = ui.GetInputAttribute("FROM");
+    p.SetInputCube(info->fileName(), inputAtt);
```

Comment:
This actually might do the same thing I mentioned in my previous comment, but in a different way. A way to check would be to use the app from the GUI instead command-line and adding an attribute. For example, if you have a multiband cube and you only want to process the first band, you would enter in the from parameter: `<cube_name>.cub+1`. I would test this with both solutions and see if you get the same output.